src/main/java/elasticsearch/concatenate/ConcatenatePlugin.java

```diff
@@ -1,20 +0,0 @@
-package elasticsearch.concatenate;
```

Comment:
File moved from `elasticsearch.concatenate` to `org.elasticsearch.plugin.analysis.concatenate` package.